support special characters in %MSYS_HOME%

yqt opened the pull request:
Characters like '(' ')' can be recognized as operators in batch, which will result in errors. e.g. msysgit is installed in "../Program Files (x86)/.."

yqt committed:
support special characters in %MSYS_HOME%

Characters like '(' ')' can be recognized as operators in batch, which will result in errors. e.g. msysgit is installed in "../Program Files (x86)/.."

davidmarble commented:
Thanks yqt! Definitely an issue that needed to be addressed. My only gripe is the change in echo statements. They were unnecessary. I prefer not to have quotation marks echoed. This is what happens on Windows:

```
C:\> echo "hi"
"hi"
```

Could you remove the changes to echo statements and re-submit? Thanks!

yqt committed:
remove quotation marks in output

use "set/p" instead of "echo" to print sentences which contain operators.
character of *backspace* helps to print 4 spaces in front of the content.

yqt committed:
remove quotation marks in output(1)

(forget to remove code for testing)
